Description: panic option rationalization
 Discard unnecessary DROP rules in NAT chain during panic.
Origin: debian
Author: Andrew Schulman <andrex@alumni.utexas.net>
Author: Phil Whineray <phil@sanewall.org>
Last-Update: 2013-03-24

--- a/firehol.sh
+++ b/firehol.sh
@@ -5930,7 +5930,9 @@
 					${IPTABLES_CMD} -t "${t}" -A "${c}" -p tcp -s "${ssh_src}" --sport "${ssh_sport}" --dport "${ssh_dport}" -m state --state ESTABLISHED -j ACCEPT
 					${IPTABLES_CMD} -t "${t}" -A "${c}" -p tcp -d "${ssh_src}" --dport "${ssh_sport}" --sport "${ssh_dport}" -m state --state ESTABLISHED -j ACCEPT
 				fi
-				${IPTABLES_CMD} -t "${t}" -A "${c}" -j DROP
+				if [ "${t}" != "nat" ] ; then
+					${IPTABLES_CMD} -t "${t}" -A "${c}" -j DROP
+				fi
 			done
 		done
 		success $"FireHOL: Blocking all communications:"
